Automattic/jetpack
# Payment blocks: Fix upgrade nudge in the site editor

Currently the upgrade nudges for payment blocks are not working in the Site Editor, on WP.com. To fix this issue, I've replaced how we insert the nudge in the editor.
Fixes https://github.com/Automattic/dotcom-forge/issues/1243
#### Changes proposed in this Pull Request:

* Wrapped the BlockEdit into a new div that contains the upgrade nudge
* Moved the nudge into the new div

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Apply the patch on your sandbox
* Get a free site and sandbox it
* Go to post-new.php and make sure that the nudge is displayed correctly for payment blocks (payment button, premium content, etc).
* Use Jetpack Beta on an Atomic site
* Get a marketplace product (this is needed so that your site remains AT after you remove the Business subscription)
* Remove the Business subscription from the site
* Repeat the tests for the blocks
